### PR TITLE
Add basic FAQ component

### PR DESCRIPTION
**Pull request by rohildshah**
Not sure about styling? Gave the questions slightly greater font weight.

### CODE REVIEW

File: src/components/shared/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -13,8 +21,10 @@ const Navbar = () => {
...
         <div className="flex align-trailing px-10 gap-[5rem]">
-          {buttonLabels.map((label, index) => (
-            <StyledNavbarButton label={label} key = {index} />
+          {Object.entries(buttonLabels).map(([key, value]) => (
+            <Link href={`/#${key}`} key={key}>
+              <StyledNavbarButton label={value} key={key} />
+            </Link>
           ))}
 
           {/* TODO: Implement color theme functionality */}
```
Comment (smmzhu) [inline src/components/shared/Navbar/Navbar.tsx:30]: Not sure what you mean by color theme functionality — feel free to write a ticket for it though

**Review by smmzhu — APPROVED** [review]
👍

File: src/components/shared/Navbar/Navbar.tsx (hunk 1 above)
Comment (rohildshah) [reply src/components/shared/Navbar/Navbar.tsx:30]: I think the dark/light mode toggle in the top right.
image

Comment (smmzhu) [reply src/components/shared/Navbar/Navbar.tsx:30]: Ah ok — think someone already created a ticket for it. But it's a non-essential feature so I'm down to table it.

**Review by dayvidwang — APPROVED** [review]
lgtm

**Pull request merged**